components/QuestionCard.js

```diff
@@ -22,11 +22,34 @@ export default function QuestionCard({ question, id, type }) {
             <textarea rows="3" maxLength="200" name={id} />
           </>
         )}
+        {type==="choice"&&(
```

Comment:
add some whitespace :)

```suggestion
        {type === "choice" && (
```